src/postprocessing/idlparsed.js

```diff
@@ -8,17 +8,170 @@
+          dfnOverload = idl.members
+            .filter(m => m.type === member.type && m.name === member.name)
+            .findIndex(m => m === member);
+        }
+        else if (member.type === 'field') {
+          dfnType = 'dict-member';
+        }
+        else {
+          dfnType = member.type;
+        }
+        if (!['constructor', 'method', 'attribute', 'enum-value', 'dict-member', 'const'].includes(dfnType)) {
+          console.error(`[error] Found unexpected IDL member type "${dfnType}" in ${spec.shortname}`);
+        }
+        dfnName = member.name ?? member.value;
+        dfnFor = idl.name;
+      }
+      else {
+        // The type of the dfn to look for is the same as the IDL type, except
+        // that composed IDL types ("interface mixin", "callback interface")
+        // only have the basic type in definitions.
+        dfnType = idl.type.split(' ')[0];
+        dfnName = idl.name;
+      }
+
+      const dfnNames = [];
+      if (dfnType === 'enum-value') {
+        // Bikeshed keeps wrapping quotes in the dfn linking text, not ReSpec.
+        dfnNames.push(dfnName);
+        dfnNames.push(`"${dfnName}"`);
+      }
+      else if (dfnType === 'method') {
+        // Bikeshed adds "..." for variadic arguments, not ReSpec. Let's try
+        // both variants. For overloads, Bikeshed essentially expects arguments
+        // to have different names, while ReSpec adds "!overload-x" to
+        // overloaded methods. We'll test all possibilities in order. If the
+        // spec only has a dfn for the most basic method, it's possible that we
+        // end up linking to that dfn from the overloaded methods too, but that
+        // seems good enough in practice.
+        // Last, method definitions sometimes appear without arguments (notably
+        // in the HTML spec).
+        const argsVariadic = member.arguments.map(arg => (arg.variadic ? '...' : '') + arg.name);
+        const args = member.arguments.map(arg => arg.name);
+        dfnNames.push(`${dfnName}!overload-${dfnOverload}(${args.join(', ')})`);
+        dfnNames.push(`${dfnName}(${argsVariadic.join(', ')})`);
+        dfnNames.push(`${dfnName}(${args.join(', ')})`);
+        dfnNames.push(`${dfnName}()`);
+      }
+      else if (dfnType === 'constructor') {
+        // Same as for methods
+        const argsVariadic = member.arguments.map(arg => (arg.variadic ? '...' : '') + arg.name);
+        const args = member.arguments.map(arg => arg.name);
+        dfnNames.push(`constructor!overload-${dfnOverload}(${args.join(', ')})`);
+        dfnNames.push(`constructor(${argsVariadic.join(', ')})`);
+        dfnNames.push(`constructor(${args.join(', ')})`);
+        dfnNames.push(`constructor()`);
+      }
+      else {
+        dfnNames.push(dfnName);
+      }
+
+      // Look for definitions that look like good initial candidates
+      const candidateDfns = spec.dfns
+        .filter(dfn => dfn.type === dfnType && !dfn.informative &&
```

Comment:
is there any reason to filter out informative definitions (e.g. do we extract wrong data without it)? 

I agree that these definitions probably ought not to be informative, but I think that would be best left to strudy rather than enforced in reffy postprocessing